airflow/api_fastapi/core_api/routes/public/dag_run.py

```diff
@@ -104,6 +106,9 @@ async def patch_dag_run_state(
                 set_dag_run_state_to_queued(dag=dag, run_id=dag_run.run_id, commit=True)
             else:
                 set_dag_run_state_to_failed(dag=dag, run_id=dag_run.run_id, commit=True)
+        elif attr_name == "note":
+            # Need to figure out how to get current user id
```

Comment:
@pierrejeambrun , in legacy implementation of update dag run note, they get current user. Kind of stuck on how to get the current user in FastAPI implementation.

Reply:
We to not have any authentication yet in the new FastAPI API, so we cannot retrieve the user.

I would suggest to include a comment there, and maybe open an issue to track that,  but for now to set the author to `None`. (The field is nullable in the database).

Reply:
Created https://github.com/apache/airflow/issues/43534

Reply:
Great thanks